lib/Parser/ptree.h

```diff
@@ -26,6 +26,7 @@ const uint fnopJump     = 0x0200;
 const uint fnopNotExprStmt = 0x0400;
 const uint fnopBinList  = 0x0800;
 const uint fnopExprMask = (fnopLeaf|fnopUni|fnopBin);
+const uint fnopAllowDefer = 0x1000; // allow to be create during defer parse
```

Comment:
>create [](start = 51, length = 6)

Nit: typo? ("allow to be created" or "allow to create")

Reply:
allow to be created.  Fixed.

---
In reply to: [178581951](https://github.com/Microsoft/ChakraCore/pull/4917#discussion_r178581951) [](ancestors = 178581951)